app/views/admins/users/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<div id="content">
+  <%= render 'admins/header' %>
```

Comment:
use new layout in admin_sessions_controller

move admins/header to layout admin